OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs

```diff
@@ -38,6 +41,9 @@ public class WithIdleOverlayInfo : PausableConditionalTraitInfo, IRenderActorPre
 		[Desc("Custom palette is a player palette BaseName")]
 		public readonly bool IsPlayerPalette = false;
 
+		[Desc("Enable to avoid UI palette modifications.")]
```

Comment:
This isn't what `Animation.IsDecoration` is for!

If you want to disable palette mods you should be specifying a palette that is excluded from the modifiers.

Reply:
This was introduced for compatibility with https://github.com/OpenRA/OpenRA/blob/86f61298e61c49b90935366d8beb071b2d1524d7/OpenRA.Mods.Common/Traits/Burns.cs#L39

Reply:
I'm asking to remove the incorrect description.

Reply:
Skimming through the code that seems to be what it is used for, excluding from `WithColorOverlay`, excluding from highlight palette effects etc. Removed the description now.